src/main/java/com/toamistaketracker/ToaMistake.java

```diff
@@ -168,9 +189,21 @@ private static String getStackingChatMessage(String message, int mistakeCount) {
         return sb.toString();
     }
 
-    private static String getAlternatingChatMessage(String message, int mistakeCount) {
-        String[] messageChoices = message.split("\\|");
+    private static String getAlternatingChatMessage(String message, String defaultMessage, int mistakeCount) {
+        String[] messageChoices = Arrays.stream(message.split("\\|")).filter(msg -> !msg.isEmpty())
+                .toArray(String[]::new);
+        if (messageChoices.length < 1) {
```

Comment:
nit: The number here should be the one that is the most logical -- in our case, we want to return the default if there are no choices (empty, or 0 length). Let's change to `messageChoices.length == 0` since it's a bit easier to read, and 0 is more recognized as a "magic number" than 1